docusaurus/docs/e2e-test-a-plugin/ci.md

```diff
@@ -82,6 +88,8 @@ To use the `version-support-policy` mode, you need to specify the `version-resol
           version-resolver-type: version-support-policy
 ```
 
-### Playwright report
+## Playwright report
+
+The end-to-end tooling generates a Playwright HTML test report for every Grafana version that is being tested. In case any of the tests fail, a Playwright trace viewer is also generated along with the report. The `Upload artifacts` step in the example workflows uploads the report as an artifact.
```

Comment:
```suggestion
The end-to-end tooling generates a Playwright HTML test report for every Grafana version that is being tested. In case any of the tests fail, a Playwright trace viewer is also generated along with the report. The `Upload artifacts` step in the example workflows uploads the report to GitHub as an artifact.
```

Maybe?